client/pages/edit-post.jsx

```diff
@@ -62,9 +63,10 @@ export default class EditPost extends React.Component {
   }
 
   handleSubmit(event) {
+    const userId = this.context.user.userId;
     event.preventDefault();
     const formData = new FormData(event.target);
-    fetch(`/api/post/${this.props.postId}/user/1`, {
+    fetch(`/api/post/${this.props.postId}/user/${userId}`, {
```

Comment:
You need to include your token in an HTTP header. This is going to require updates to your back end routes. You should use `req.user` and not `req.params` to determine who is sending the request.

The `fetch()` call will need an `'X-Access-Token'` header.

```js
fetch(`/api/post/${this.props.postId}`, {
  method: 'PUT',
  headers: { 
    'X-Access-Token': thetoken
  },
  body: formData
})
```